common/src/main/java/com/wynntils/screens/maps/MainMapScreen.java

```diff
@@ -212,6 +212,21 @@ protected void doInit() {
                                 .append(Component.translatable("screens.wynntils.map.waypoints.add.name")),
                         Component.translatable("screens.wynntils.map.waypoints.add.description")
                                 .withStyle(ChatFormatting.GRAY))));
+
+        if (firstInit) {
+            BoundingBox textureBoundingBox =
+                    BoundingBox.centered(mapCenterX, mapCenterZ, width / currentZoom, height / currentZoom);
+
+            // When in an unmapped area, center to the middle of the map if the feature is enabled
+            if (Managers.Feature.getFeatureInstance(MainMapFeature.class)
+                            .centerWhenUnmapped
+                            .get()
+                    && Services.Map.getMapsForBoundingBox(textureBoundingBox).isEmpty()) {
```

Comment:
Didn't you add a method for `isPlayerInMappedArea`? Use that here.

Reply:
Actually I don't know if that is what we want to do.. Although it is weird that the same config works differently on the guild map.